core/domain/user_services.py

```diff
@@ -36,7 +36,7 @@ class UserSettings(object):
     def __init__(
             self, user_id, email, username=None, last_agreed_to_terms=None,
             last_started_state_editor_tutorial=None,
-            profile_picture_data_url=None, user_bio='',
+            profile_picture_data_url=None, user_bio='', subject_interests=[],
```

Comment:
Don't do this -- it's a common Python gotcha. Use None instead and replace with [] if it's really None.

http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments